runtime/src/lib.rs

```diff
@@ -80,7 +81,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // implementation changes and behavior does not, then leave spec_version as
     // is and increment impl_version.
     spec_version: 226,
-    impl_version: 1,
+    impl_version: 2,
```

Comment:
I’d say this constitutes a spec version change?

Reply:
I think so, since the `construct_runtime!` is `pub` and has a new field: `Identity`